AddressService/AddressService.Core/Utils/CompressionUtils.cs

```diff
@@ -42,13 +56,26 @@ public static string Unzip(byte[] bytes)
             {
                 using (var gs = new GZipStream(msi, CompressionMode.Decompress))
                 {
-                    //gs.CopyTo(mso);
```

Comment:
can we get rid of this comment

Reply:
the comment is removed in the latest comment already